docs/advanced/command_line.md

```diff
@@ -29,21 +59,29 @@ On Windows, there are two executable files for QuPath.
 It is necessary to use *"QuPath (console).exe"* here to be able to view the output, e.g.
 
 ```bash
-"QuPath-0.2.0 (console).exe" --help
+"QuPath-0.4.3 (console).exe" --help
 ```
 
 :::{figure} images/command_line_win.png
 :align: center
 :class: shadow-image
 :width: 80%
 :::
+&NewLine;
+
+Also, when using the *PowerShell* command line interface instead of *Command Prompt*, the QuPath executable name must be preceded with an ampersand otherwise parser errors such as `ParentContainsErrorRecordException` may occur.
+See [this forum post](https://forum.image.sc/t/running-qupath-using-command-line-interface/72518/6).
+
+```bash
+& 'QuPath-0.4.3 (console).exe' --help
+```
 
 ### Mac
 
 On macOS, the executable is buried inside the `.app` file and therefore you need something like:
 
 ```bash
-./QuPath-0.2.0.app/Contents/MacOS/QuPath-0.2.0 --help
+./QuPath-0.4.3.app/Contents/MacOS/QuPath-0.4.3 --help
```

Comment:
> not sure if there was previously?

Quick sanity check, yes it's still written like this on [readthedocs](https://qupath.readthedocs.io/en/0.4/docs/advanced/command_line.html#command-line).

If there's no version number associated with the executable, that's an easy fix, thanks!

Reply:
Thanks, yeah the docs definitely had the number before - I'm just not sure if the executable including the version (it maybe did, because the packager changed over the years).

Anyhow, it was wrong for v0.4 and now it's correct so thanks!